### PR TITLE
show edit&cancel btns only for renter's pending status

### PR DESCRIPTION
**Pull request by joycehwchan**
Screen Shot 2022-11-18 at 8 58 28

### CODE REVIEW

**Comment by joycehwchan** [comment]
fixed flex with separate form btns

Screen Shot 2022-11-18 at 9 11 33
Screen Shot 2022-11-18 at 9 11 39

**Comment by twk-mn** [comment]
Is there supposed to be an empty class="" around the booking confirm/reject buttons?

File: app/views/bookings/_card.html.erb [hunk 1]
```diff
@@ -23,16 +23,20 @@
...
+            <div class="btn-group-booking-landowner">
               <% if booking.status == 'pending' %>
-                <%= simple_form_for booking, url: landowner_booking_path(booking) do |f| %>
-                  <%= f.input :status, as: :hidden, input_html: { value: 'confirmed'} %>
-                  <%= f.submit 'Confirm booking', class: 'btn btn-primary booking-btn' %>
-                <% end %>
-                <%= simple_form_for booking, url: landowner_booking_path(booking) do |f| %>
-                  <%= f.input :status, as: :hidden, input_html: { value: 'rejected'} %>
-                  <%= f.submit 'Reject booking', class: 'btn btn-outline-danger booking-btn' %>
+                <div class="">
```
Comment (twk-mn) [inline app/views/bookings/_card.html.erb:28]: Should this be empty? @joycehwchan

Comment (joycehwchan) [reply app/views/bookings/_card.html.erb:28]: This div doesn't need any classes. Gonna remove that, thanks.

**Pull request merged**